app/controllers/idv/verify_info_controller.rb

```diff
@@ -392,11 +392,23 @@ def add_proofing_costs(results)
         elsif stage == :threatmetrix
           # transaction_id comes from request_id
           tmx_id = hash[:transaction_id]
+          log_irs_tmx_fraud_check_event(hash) if tmx_id
           add_cost(:threatmetrix, transaction_id: tmx_id) if tmx_id
         end
       end
     end
 
+    def log_irs_tmx_fraud_check_event(result)
+      return unless FeatureManagement.proofing_device_profiling_collecting_enabled?
```

Comment:
Would it make sense to add your new methods to [ThreatMetrixStepHelper](https://github.com/18F/identity-idp/blob/main/app/services/idv/steps/threat_metrix_step_helper.rb), which is already included in the SsnStep and could be added to VerifyInfoController? Also do we want to add anything about this to the [VerifyInfo controller spec](https://github.com/18F/identity-idp/blob/main/spec/controllers/idv/verify_info_controller_spec.rb)? Happy to pair on any of that if it would be helpful.

Reply:
I went ahead and fixed the merge conflict from this, since I knew where it came from.

Reply:
Thank you! i will push Rspec for [VerifyInfo controller spec](https://github.com/18F/identity-idp/blob/main/spec/controllers/idv/verify_info_controller_spec.rb) soon.

Reply:
Tests look good! I made one note about No Results.